### PR TITLE
Document build process for new package structure

### PR DESCRIPTION
**Pull request by ChristianSi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,17 +104,30 @@ But it's best to use ghc 7.8.x, where x >= 4. ghc 7.10 is not yet
 supported. If your package manager doesn't have a suitable ghc version, you
 can download it manually from https://www.haskell.org/ghc/.
 
-Clone the Thentos repository from GitHub. Afterwards create a cabal sandbox
-in the same directory and install Thentos:
+Clone the Thentos repository from GitHub. There are several package in
```
Comment (fhartwig) [inline README.md]: s/package/packages

File: README.md [hunk 2]
```diff
@@ -104,17 +104,30 @@ But it's best to use ghc 7.8.x, where x >= 4. ghc 7.10 is not yet
...
+* `thentos-core`: the core package
+* `thentos-tests`: tests for `thentos-core`
+* `thentos-adhocracy`: integration with the Adhocracy software
+* ...
+
+You'll need to build `thentos-core` in any case. `thentos-tests` is only
+required if you want to run tests. Whether you need other packages will
+depend on your use case.
+
+We recommend building all required package into the same sandbox. To
```
Comment (fhartwig) [inline README.md]: s/package/packages

File: README.md [hunk 3]
```diff
@@ -104,17 +104,30 @@ But it's best to use ghc 7.8.x, where x >= 4. ghc 7.10 is not yet
...
+Install `thentos-core`, change into the `thentos-core` directory and
```
Comment (fhartwig) [inline README.md]: s/Install/install

File: README.md [hunk 4]
```diff
@@ -104,17 +104,30 @@ But it's best to use ghc 7.8.x, where x >= 4. ghc 7.10 is not yet
...
+execute the following commands:
 
 ```shell
-$ cabal sandbox init
-$ cabal install --enable-tests --enable-documentation --dependencies-only
+$ cabal sandbox init --sandbox=../.cabal-sandbox
```
Comment (fhartwig) [inline README.md]: We should probably mention that you need to `cabal sandbox init` in the repo root first.

**Comment by fhartwig** [comment]
A few minor comments, but :+1: otherwise.

File: README.md (hunk 4 above)
Comment (fisx) [inline README.md]: I don't think you have to do that.  cabal will just create the sandbox there if it does not exist.

Comment (fhartwig) [inline README.md]: Oh, neat. Sorry for the noise.

**Pull request merged**